src/OrchardCore/OrchardCore.DisplayManagement/Razor/RazorPage.cs

```diff
@@ -51,6 +54,14 @@ private void EnsureShapeFactory()
             }
         }
 
+        private void EnsureShellFeaturesManager()
```

Comment:
Remove?

Reply:
> I don't like it, TheAdmin has a dependency on the other feature. The feature should inject the shape.

There's no dependency, I'm still looking for a better way to inject shape without throws when the feature is disabled